soda/cmd/generate/model.go

```diff
@@ -214,20 +215,30 @@ func (m model) GenerateSQLFromFizz(content string, f fizz.Translator) string {
 	return content
 }
 
-func newModel(name string) model {
+func newModel(name string, structTag string) (model, error) {
 	m := model{
 		Package:               "models",
 		Imports:               []string{"time", "github.com/gobuffalo/pop", "github.com/gobuffalo/validate"},
 		Name:                  nflect.New(name),
 		Attributes:            []attribute{},
 		ValidatableAttributes: []attribute{},
 		attributesCache:       map[string]struct{}{},
+		StructTag:             structTag,
+	}
+
+	switch structTag {
+	case "json":
+		m.Imports = append(m.Imports, "encoding/json")
+	case "xml":
+		m.Imports = append(m.Imports, "encoding/xml")
+	default:
+		return model{}, errors.New("invalid struct tags (use xml or json)")
 	}
 
 	m.addAttribute(attribute{Name: flect.New("created_at"), OriginalType: "time.Time", GoType: "time.Time", PreventValidation: true})
```

Comment:
Should probably pass structTag to these as well since you are by passing using `newAttribute`.

Reply:
You're right. :)

Reply:
@mclark4386 Fixed.

Reply:
@stanislas-m thanks! sorry about the delay!